src/Components/Politicians/PoliticianCard/index.js

```diff
@@ -8,14 +8,15 @@ import * as styles from './styles';
 
 export type Props = {
   politician: PoliticianType,
+  style?: InlineStyle,
```

Comment:
Did `Object` not work here?
I just get concerned when using libraries that are not often used

Reply:
this might not be relevant with the comments made in `VoteRecord`